devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy

```diff
@@ -21,16 +21,18 @@ metadata {
 		capability "Battery"
 		capability "Configuration"
 		capability "Refresh"
-		capability "Window Shade"
-		capability "Health Check"
+        capability "Health Check"
+		capability "Window Shade"		
+        capability "Switch"
+        capability "Switch Level"
 
 		attribute("replay", "enum")
 		attribute("battLife", "enum")
 
-		//command "pause"
+		command "pause"
```

Comment:
`pause` is a command in the `Window Shade` capability so this custom command should be removed

Reply:
I have now fixed this.